### DISCUSSION TITLE
Wrong location with the event API

### DISCUSSION THREAD

**Discussion by mquandalle** [plausible/analytics#2267] [Anything Else]
I'm using the [event API](https://plausible.io/docs/events-api) to send a custom event from a server. As documented I'm following the IP address of the user with the `X-Forwarded-For` header but it seems that the location inferred by Plausible (country / city) is based on the IP of my server, not the one of my user provided through the HTTP header.

Is it a misconfiguration on my side or a problem with Plausible?

**Comment by RobertJoonas** [comment]
Hi @mquandalle!

We use https://db-ip.com for getting visitor locations. The locations we show only depend on the IP addresses from the `X-Forwarded-For` header. So I don't think it's a bug on our end. Could you please write to our support at support@plausible.io, and share some implementation details? We can then have a closer look and try to debug the issue.

Thank you!
